app/src/test/java/com/gholem/moneylab/features/createNewCategory/viewmodel/CreateNewCategoryViewModelTest.kt

```diff
@@ -29,9 +29,11 @@ class CreateNewCategoryViewModelTest {
 
     private val insertCategoryModelUseCaseMock: InsertCategoryModelUseCase =
         Mockito.mock(InsertCategoryModelUseCase::class.java)
-    private val navigationMock: NavigationLiveData<CreateNewCategoryEvent> =
-        Mockito.mock(NavigationLiveData::class.java)
-                as NavigationLiveData<CreateNewCategoryEvent>
+    private val navigationMock: NavigationLiveData<CreateNewCategoryNavigationEvent> =
```

Comment:
Refactor navigation tests in the same way as in `AddTransactionViewModelTest` in all ViewModel tests (remember also about changing the `navigation` from `var` to `val` in every VM)